### ISSUE TITLE
Adjust error messages for `golden-test verify`

### ISSUE THREAD

**Issue by drevell** [abcxyz/abc#393]
### TL;DR

We can make this a little more user-friendly.

As a user, when I run the `golden-test verify` command, if the test.yaml exists but the `data` directory doesn't exist, then we should suggest running the `record` command to create the `data` directory. Something like `there's no recorded golden output for this test, you might need to run the "record" command to create it."

As a user, when I run the `golden-test verify` command, if the verify command fails because the actual output doesn't match the expected output, then we should suggest that they should consider running the `record` command if the new output working as intended and should replace the previous golden output. Something like `if the new template output is correct, you might consider running the "record" command to capture it as the new expected output.`
